doc/man1/openssl-pkcs12.pod.in

```diff
@@ -237,23 +237,33 @@ the use of signing only keys for SSL client authentication.
 
 Specify the MAC digest algorithm. If not included them SHA1 will be used.
 
-=item B<-nomaciter>, B<-noiter>
+=item B<-iter> I<count>
 
-These options affect the iteration counts on the MAC and key algorithms.
-Unless you wish to produce files compatible with MSIE 4.0 you should leave
-these options alone.
+This option specifies the encryption key iteration count.
 
 To discourage attacks by using large dictionaries of common passwords the
 algorithm that derives keys from passwords can have an iteration count applied
 to it: this causes a certain part of the algorithm to be repeated and slows it
-down. The MAC is used to check the file integrity but since it will normally
+down. The default value is 2048.
+
+=item B<-noiter>
+
+Don't use encryption key iteration. Equivalent to B<-iter 1>.
```

Comment:
Ping @t8m or @levitte should that ```B<>``` be ```C<>```?  Or maybe just say "an interation of one" and avoid the whole mess.

Reply:
I think it should be `C<-iter 1>`

Reply:
C